tests/unit/serializers/tumblr-post-test.js

```diff
@@ -1,3 +1,4 @@
+import Ember from 'ember';
```

Comment:
'import' is only available in ES6 (use esnext option).<br>'Ember' is defined but never used.